python/pyiceberg/catalog/hive.py

```diff
@@ -229,24 +290,43 @@ def create_table(
             ValueError: If the identifier is invalid
         """
         database_name, table_name = self.identifier_to_database_and_table(identifier)
-        current_time_millis = int(time.time())
+        seconds_since_epoch = int(time.time())
```

Comment:
The table metadata requires millis, but this produces seconds. Can you change this to set the table metadata correctly?

```python
current_time_millis = int(time.time() * 1000)
```

Reply:
Thanks, this was because Hive needs seconds.